examples/imgui_impl_win32.cpp

```diff
@@ -419,15 +419,15 @@ typedef DPI_AWARENESS_CONTEXT(WINAPI * PFN_SetThreadDpiAwarenessContext)(DPI_AWA
 
 void ImGui_ImplWin32_EnableDpiAwareness()
 {
-    // if (IsWindows10OrGreater()) // FIXME-DPI: This needs a manifest to succeed. Instead we try to grab the function pointer.
+    /* if (IsWindows10OrGreater()) // FIXME-DPI: This needs a manifest to succeed. Instead we try to grab the function pointer.
     {
```

Comment:
Note: need an explanation about why that chunk of `ImGui_ImplWin32_EnableDpiAwareness()` was commented. You said you ran into a problem with it, details would be good.